mava/utils/jax_training_utils.py

```diff
@@ -18,7 +18,7 @@ def action_mask_categorical_policies(
     return tfd.Categorical(logits=masked_logits, dtype=distribution.dtype)
 
 
-def set_growing_gpu_memory() -> None:
+def set_growing_gpu_memory_jax() -> None:
```

Comment:
Is there a reason for the "jax" at the end?

Reply:
There is a jax and tf version of this function. Both set diff env vars.

Reply:
Do we still set the tf version? As we still use tf functions? It does not seem that we have a GPU memory problem in your PR so maybe you fixed it in some other way?

Reply:
Yep. We still have the tf one and call both in our trainers.